addons/hide-stage/userscript.js

```diff
@@ -1,34 +1,76 @@
 export default async function ({ addon, console, msg }) {
+  // Wait until the project is loaded so that the renderer will definitely exist.
+  const vm = addon.tab.traps.vm;
+  await new Promise((resolve) => {
+    if (vm.editingTarget) return resolve();
+    vm.runtime.once("PROJECT_LOADED", resolve);
+  });
+
   let stageHidden = false;
-  let bodyWrapper;
   let smallStageButton;
   let largeStageButton;
   let hideStageButton;
 
   function hideStage() {
     stageHidden = true;
-    if (!bodyWrapper) return;
-    bodyWrapper.classList.add("sa-stage-hidden");
+    document.documentElement.classList.add("sa-stage-hidden");
```

Comment:
We could use the body instead AFAIK. Not necessary to change it tho, root element is fine too